src/main/kotlin/lottery/view/ResultView.kt

```diff
@@ -16,11 +16,11 @@ object ResultView {
         lotteries.map { println(it.toString()) }
     }
 
-    fun printMatchNumbers(start: Int, end: Int, match: RankCounts) {
-        (start..end).forEach { printMatchNumber(match.retrieve(Rank.valueOf(it)), Rank.valueOf(it)) }
+    fun printMatchNumbers(match: RankCounts) {
+        Rank.values().forEach { printMatchNumber(match.retrieve(it), it) }
     }
 
     private fun printMatchNumber(matchCount: Int, rank: Rank) {
-        println("${rank.matchCount}개 일치 (${rank.price}원)- $matchCount")
+        println("${RankPrinter.valueOf(rank).content} (${rank.price}원)- $matchCount")
```

Comment:
```suggestion
        println("${rank.matchCount}개 일치 (${rank.price}원)- $matchCount")
```
이렇게 해주면 RankPrinter가 없어도 괜찮을 것 같아요 :)

Reply:
상세한 피드백 감사합니다.! 🙇 

말씀해주신 부분처럼 작업을 하려고 했으나 보너스볼인 경우 `보너스 볼 일치`라는 내용이 추가되어야 되기 때문에 별도의 조건문이 작성되어야 한다고 생각을 하였습니다.
또한 만약 보너스볼처럼 추가적인 출력형태가 요구되면 `printMatchNumber` 가 수정되어야 한다고 생각했습니다.
따라서, `printMatchNumber`가 출력형태가 변경될 때 뿐만 아니라 새로운 Rank가 추가될 때도 변경이 이뤄저야 하기 때문에
2가지의 일을 할 것이라고 생각을 했기 때문에 SRP도 위배되고 Rank 추가시 출력이라는 핵심 기능이 변경이 되어야 하기 때문에 OCP 에 위배가 된다고 생각을 하였습니다.

따라서 RankPrinter 가 존재한다면 출력문이 변경될 때는 `printMatchNumber` 가 변경되고 별도의 출력형태의 Rank 가 추가된다면 `RankPrinter` 만 수정되면 된다고 생각하여서 `RankPrinter`를 추가하게 되었습니다. 🙋

혹시 이것에 대해서 제가 잘못 생각하고 있는 부분이 있다면 지적 및 피드백 부탁드리겠습니다.
감사합니다.! 😃